### DISCUSSION TITLE
Not able to install laravel v8.66.0

### DISCUSSION THREAD

**Discussion by ghost** [laravel/framework#44043] [Q&A]
I'm trying to install Laravel v8.66.0 and I can't.

```
composer create-project laravel/laravel:^8.66.0 summernote
composer create-project laravel/laravel:8.66.0 summernote
composer create-project --prefer-dist laravel/larabel summernote "v8.66.0"
composer create-project --prefer-dist laravel/larabel summernote "8.66.0"
composer global require laravel/installer "v8.66.0"
composer global require laravel/installer "8.66.0"
```

All theese comands fail with:
```
[InvalidArgumentException]
  Could not find package laravel/laravel with version v8.66.0.
```

**Comment by henzeb** [comment]
Weird, it should exists. https://github.com/laravel/framework/releases/tag/v8.66.0

Try without the trailing 0?

**Reply by ghost** [reply]
Yes, it is really weird. I saw this version in github, and I have it installed in other laravel projects, I don't understand.

I tested it, with same error:
```
C:\xampp\htdocs>composer create-project laravel/laravel:8.66 summernote
Creating a "laravel/laravel:8.66" project at "./summernote"
Info from https://repo.packagist.org: #StandWithUkraine


  [InvalidArgumentException]
  Could not find package laravel/laravel with version 8.66.
```

**Reply by henzeb** [reply]
You miss a v there. Try with the v.

**Comment by crynobone** [comment]
https://packagist.org/packages/laravel/laravel#v8.6.12

**Reply by ghost** [reply]
Thanks you, but that is other version. I need **v8.66.0**:
            "name": "laravel/framework",
            "version": "v8.66.0",
            "source": {
                "type": "git",
                "url": "https://github.com/laravel/framework.git",
                "reference": "b757fe143a3ab802a08a191032fc7046b582f107"
            },
            "dist": {
                "type": "zip",
                "url": "https://api.github.com/repos/laravel/framework/zipball/b757fe143a3ab802a08a191032fc7046b582f107",
                "reference": "b757fe143a3ab802a08a191032fc7046b582f107",
                "shasum": ""
            }

**Reply by crynobone** [reply]
But your using `composer create-project "laravel/laravel"` and not `"laravel/framework"`. Notice the issue there?

**Reply by ghost** [reply]
Oh, I'm not familiar with composer. 
I have laravel project with this version:
```
php artisan --version
Laravel Framework 8.66.0
```

And I need install the same laravel version with composer. I see no "laravel/laravel" in composer.lock in previous project.

**Reply by henzeb** [reply]
The installation/creation is done with laravel/laravel. The laravel version in packagist is there, without the trailing zero.

https://packagist.org/packages/laravel/laravel#v8.6.6

**Reply by ghost** [reply]
With this?
`composer create-project laravel/laravel:8.6.6 summernote`

it  installed other version
```
C:\xampp\htdocs\summernote>php artisan --version
Laravel Framework 8.83.23
```
